generators/cli/index.js

```diff
@@ -21,7 +21,7 @@ module.exports = generators.Base.extend({
       this.fs.copyTpl(
         this.templatePath('cli.js'),
         this.destinationPath('lib/cli.js'), {
-          pkgSafeName: _.camelCase(this.options.name),
+          pkgSafeName: _.camelCase(this.appname),
```

Comment:
Actually using `--name` work. I wasn't aware of that, my mistake. I did not provide and it still ran without errors, I guess some check for that might be useful.